tests/openwisp2/asgi.py

```diff
@@ -12,12 +13,18 @@
     from .sample_notifications import consumers
 
     application = ProtocolTypeRouter(
-        {'websocket': AuthMiddlewareStack(URLRouter(get_routes(consumers)))}
+        {
+            "http": get_asgi_application(),
```

Comment:
I don't think `http` will be needed. We are not handling HTTP requests with ASGI application.

Reply:
Okay, I will remove it.